Updating uninstallation instructions for LambdaTest (#1599)
DataDog/integrations-extras: lambdatest/README.md

### Uninstallation

> Once you uninstall this integration, any previous authorizations will be revoked. 

Reviewer: ```suggestion
Once you uninstall this integration, any previous authorizations are revoked. 
```

Author: Yes this works! Thanks
